test/unit/loudoutInventroy.spec.js

```diff
@@ -0,0 +1,397 @@
+import { assert } from 'chai';
+import { describe, it } from 'mocha';
+
+import LoadOutInventory from '../../src/LoadOutInventory.js';
+
+describe('LoudOutInventory', () => {
+  describe('#contructor', () => {
+    it('handle some data', () => {
+      const data = {
```

Comment:
could we stick this in a datafile so it's not so much mocked data to sift through to get to the test?

suggestion: `loadoutInventory.data.js`

also, `Inventory` is misspelled `Inventroy` in your filename